components/region_cache_memory_engine/src/gc.rs

```diff
@@ -0,0 +1,481 @@
+// Copyright 2023 TiKV Project Authors. Licensed under Apache-2.0.
+
+use core::slice::SlicePattern;
+use std::{fmt::Display, sync::Arc};
+
+use engine_traits::{CF_DEFAULT, CF_WRITE};
+use skiplist_rs::Skiplist;
+use slog_global::{info, warn};
+use txn_types::{Key, WriteRef, WriteType};
+
+use crate::{
+    keys::{decode_key, encoding_for_filter, InternalKey, InternalKeyComparator},
+    RegionCacheMemoryEngine,
+};
+
+fn split_ts(key: &[u8]) -> Result<(&[u8], u64), String> {
```

Comment:
```suggestion
fn extract_key_ts(key: &[u8]) -> Result<(&[u8], u64), String> {
```

Reply:
It follows the name in compaction_filter.rs

Reply:
add a comment:
```
/// Extract key and timestamp. See also: `split_ts` in `compaction_filter.rs`.